.github/workflows/test.yml

```diff
@@ -16,22 +16,15 @@ jobs:
       - name: Check out repository code
         uses: actions/checkout@v4
 
-      - name: Install poetry
-        run: pipx install poetry==1.2.2
-
-      - name: Install python or load from cache with dependencies
+      - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: "3.9"
-          cache: poetry
+          python-version: "3.10"
 
       - name: Install dependencies
         run: |
+          python -m pip install poetry
           poetry install
 
       - name: Run test suite
-        run: |
-          poetry run python -m pytest \
-          --nbmake \
-          --nbmake-find-import-errors \
-          --nbmake-timeout=20 -vvv
+        run: make test
```

Comment:
**explain:** imo it's good to run the tests the same way, locally and in CI, though I understand that they don't have to always be 1:1, so I can revert this.

Reply:
No this looks good!